features/url-schemes.html

```diff
@@ -14,6 +14,8 @@ <h1>URL Schemes</h1>
  <li><a href="ddgQuickLink://example.com">ddgQuickLink://example.com</a> - launches DuckDuckGo app and opens the given URL</li>
  <li><a href="zoomus://open/1231231231">zoomus://open/1231231231</a> - should launch the zoom app, but is not a valid zoom link</li>
  <li><a href="custom://scheme/with.path">custom://scheme/with.path</a> - a custom URL scheme that probably isn't supported</li>
+ <li><a href="intent://duckduckgo.com/?q=test#Intent;scheme=https;package=com.duckduckgo.mobile.android;S.browser_fallback_url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.duckduckgo.mobile.android;end">intent://duckduck.com</a> - launches DuckDuckGo Android app using a query parameter or opens the PlayStore</li>
+ <li><a href="intent://duckduckgo.com#Intent;scheme=https;package=com.duckduckgo.mobile.android;S.browser_fallback_url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.duckduckgo.mobile.android;end">intent://duckduck.com</a> - launches DuckDuckGo Android app or opens the PlayStore</li>
```

Comment:
Not an issue with the `href`, but should visible text be `intent://duckduckgo.com` to match?

```diff
- intent://duckduck.com
+ intent://duckduckgo.com
```